docs/JSON-RPC.md

```diff
@@ -29,6 +29,8 @@ The JSON-RPC server is based on the [JSON-RPC 2.0](https://www.jsonrpc.org/speci
 - A **response** from Jamulus to the consumer.
 - A **notification** from Jamulus to the consumer.
 
+**Tip:** On Linux, you can connect to a JSON-RPC server using the `nc` CLI tool. On Windows, you can download and use the `ncat` CLI tool.
```

Comment:
It might be worth add "for scripting and dianostic purposes" - as, more general use, these tools probably aren't what you'd be using.  You'd probably be writing a program.

Reply:
I have added a link to a Node.js snippet.